utilities/nodes/map_editor/README.md

```
@@ -56,3 +56,45 @@ Now that you have a perfectly closed loop in your `MAP_NAME.png`, we feed it bac
 
 
 You are now fully configured and ready to race! Verify by running the base system **on the car** with the new map.
+
+# Mapping from a Bag
+
+## Step 1: Generate the bag
+
+1. First, you must attach a second container, and cd into the data folder
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)